client/blocks/upload-image/docs/example.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { partial } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import UploadImage from '../';
+import { AspectRatios } from 'state/ui/editor/image-editor/constants';
+
+export default class UploadImageExample extends Component {
+	state = {
+		isUploading: false,
+		uploadedImageDataUrl: null
```

Comment:
Sorry, I mean that we aren't using this state in any place. It's cool see how its value is propagated through. Feel free to ignore the comment above.